src/otx/core/model/entity/detection.py

```diff
@@ -143,3 +142,67 @@ def _customize_outputs(
             bboxes=bboxes,
             labels=labels,
         )
+
+
+class OVDetectionCompatibleModel(OTXDetectionModel):
+    """Object detection model compatible for OpenVINO IR inference.
+
+    It can consume OpenVINO IR model path or model name from Intel OMZ repository
+    and create the OTX detection model compatible for OTX testing pipeline.
+    """
+
+    def __init__(self, config: DictConfig) -> None:
+        self.model_name = config.pop("model_name")
+        self.config = config
+        super().__init__()
+
+    def _create_model(self) -> nn.Module:
+        from openvino.model_api.models import DetectionModel
+
+        return DetectionModel.create_model(self.model_name)
+
+    def _customize_outputs(
+        self,
+        outputs: Any,  # noqa: ANN401
+        inputs: DetBatchDataEntity,
+    ) -> DetBatchPredEntity | OTXBatchLossEntity:
+        # add label index
+        outputs_objects = outputs.objects
+        bboxes = [
```

Comment:
This should be `tv_tensor.BoundingBoxes`.

Reply:
It can work perfectly without tv_tensor.BoundingBoxes. However, for the types consistency I can change it

Reply:
> However, for the types consistency I can change it
Yes, please keep it consistent and care about the runtime type mismatching.

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2748/commits/fa36076ef72b7ad990a68b70db33ed4eb007a889)